recipes/chemfp/meta.yaml

```diff
@@ -24,12 +25,12 @@ requirements:
   build:
     - {{ compiler('c') }}
   host:
-    - python
+    - python <=3
```

Comment:
Smaller or equal 3? So we can remove this, left?

Reply:
No because this version still requires python 2.*.

Or am I missing what you're saying here?

Reply:
If so the selector should be `<3` I think.

Reply:
Oh I get it.